src/main/java/org/jenkinsci/plugins/aquadockerscannerbuildstep/ScannerExecuter.java

```diff
@@ -71,11 +77,23 @@ public static int execute(Run<?, ?> build, FilePath workspace, Launcher launcher
 				}
 				break;
 			case "local":
+				if(!scannerPath.equals("") && !containerRuntime.equals("docker")) {
```

Comment:
can we use one boolean variable for this !containerRuntime.equals("docker"). And use that boolean at all places
Boolean isDocker := true
if !containerRuntime.equals("docker")
{
     isDocker=false;
}